AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/QtPyO3DEEditor.py

```diff
@@ -90,3 +92,12 @@ def close_asset_editor(self) -> None:
                                                WAIT_TIME_SEC_3)
 
         assert result, "Failed to close Asset Editor"
+
+    def make_new_editor_entity(self, entity_name: str, component_list: list, position=math.Vector3(512.0, 512.0, 32.0)) \
+            -> EditorEntity:
+
+        editor_entity = EditorEntity.create_editor_entity_at(position, entity_name)
+        editor_entity.add_components(component_list)
+
+        return editor_entity
+
```

Comment:
This doesn't belong in `QtPy`. Using `EdtiorEntity Utils` to do validations is fine, but using the utils to perform actions to be validated shouldn't be done and causes code duplication.

`QtPy` is a last step for performing actions when not offered by `azlmbr` or the extended `EditorEntity Utils`.

Reply:
this function is removed from the qtpy class